### PR TITLE
Enhancement

### PR DESCRIPTION
**Pull request by sanjevscet**
Restify Sample code is updated by  adding Typrorm Feature, Passport Authentication, Yup Validation, all server routes are prefixed by 'api', creating env file reader & many more.

 As per my knowledge, I have added all the main code snippets.

### CODE REVIEW

File: app/helpers/env.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        const JWT_SECRET = process.env.JWT_SECRET || 'randomText';
+        const passportStrategy: StrategyOptions = {
+            jwtFromRequest: ExtractJwt.fromAuthHeaderWithScheme(JWT_AUTH_HEADER),
+            secretOrKey: JWT_SECRET,
+        };
+
+        const PER_PAGE = 100;
+
+        // APP Meta Data
+        const APP_NAME = process.env.APP_NAME || 'BYJUS';
```
Comment (sn123) [inline app/helpers/env.ts]: change this to something like 'example'

File: app/helpers/env.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        const ENVIRONMENT = process.env.ENVIRONMENT || 'LOCAL';
+        const BUGSNAG_API_KEY = process.env.BUGSNAG_API_KEY || 'abc'; // set some value i.e. abc otherwise server will be crashed
```
Comment (sn123) [inline app/helpers/env.ts]: change it to '<your-key-here>'

Comment (sn123) [inline app/helpers/env.ts]: shouldn't this be read only if bugsnag is enabled?

File: bootstrap/bootstrapper.ts [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        }
+        this.bootstrapWorker();
+    }
+
+    private static bootstrapWorker(): void {
+        new Promise(
+            async (resolve, reject): Promise<void> => {
+                try {
+                    Bugsnag.start({
+                        apiKey: 'ef38e6e76834f9ebb8a0bfbc30621b3f',
```
Comment (sn123) [inline bootstrap/bootstrapper.ts]: why is this hard-coded here?

**Review by sn123 — CHANGES_REQUESTED** [review]

File: bootstrap/bootstrapper.ts (hunk 3 above)
Comment (sanjevscet) [reply bootstrap/bootstrapper.ts]: api is removed from code & key is re-generated

File: app/helpers/env.ts (hunk 2 above)
Comment (sanjevscet) [reply app/helpers/env.ts]: done

File: app/helpers/env.ts (hunk 1 above)
Comment (sanjevscet) [reply app/helpers/env.ts]: done

**Review by sn123 — APPROVED** [review]

**Pull request merged**